protocols/request-response/src/lib.rs

```diff
@@ -58,31 +58,15 @@
 
 #![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
 
-mod codec_priv;
-#[deprecated(
-    note = "The `codec` module will be made private in the future and should not be depended on."
-)]
-pub mod codec {
-    pub use super::codec_priv::*;
-}
-
-mod handler_priv;
-#[deprecated(
-    note = "The `handler` module will be made private in the future and should not be depended on."
-)]
-pub mod handler {
-    pub use super::handler_priv::*;
-}
+pub mod codec;
+pub mod handler;
```

Comment:
```suggestion
mod codec;
mod handler;
```

We want these to be private!